codes/python/chapter_stack_and_queue/array_stack.py

```diff
@@ -4,7 +4,78 @@
 Author: Krahets (krahets@163.com)
```

Comment:
Please update the author and time information in the file header with yours, thanks!